README.md

```diff
@@ -15,14 +15,14 @@ one) to recover from downtime quickly. (See the docs for `NewFrom()` for caveats
 
 ### Installation
 
-`go get github.com/patrickmn/go-cache`
+`go get github.com/youjianglong/go-cache`
```

Comment:
Seriously?!

Reply:
Sorry, I didn't notice this. I'll submit it.